.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: check-yaml
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v2.7.1"
+    rev: "v3.0.0-alpha.0"
```

Comment:
@henryiii do we want alpha releases?

Reply:
It doesn't break anything, so I've generally not bothered trying to fix it.

Reply:
@henryiii Sorry, but does "It doesn't break anything" mean "v3.0.0-alpha.0 doesn't break anything and so there's no problem and we can merge this in" or does it mean "v2.7.1 isn't a problem so there's no need to accept an alpha release"?

Reply:
The former, I've just been allowing it to be merged since the "release" seems stable enough and hasn't changed anything formatting-wise that I can see.

Reply:
Cool then I'll merge this.